openapi/components/schemas/PaymentInstrumentUpdateToken.yaml

```
@@ -12,10 +12,15 @@ properties:
     description: |-
       ID of the sticky gateway account.
       All future payments are processed by this gateway account.
+      To remove sticky gateway account from payment instrument, set this value to `null`.
```

suggestion: for readability
```suggestion
      To remove the sticky gateway account from a payment instrument, set this value to `null`.
```